code/modules/mob/living/carbon/human/emote.dm

```diff
@@ -236,6 +236,8 @@
 				msg = "<B>[src]</B> has left the building..."
 			if(M_HARDCORE in mutations)
 				msg = "<B>[src]</B> whispers with \his final breath, <i>'i told u i was hardcore..'</i>"
+				if(miming)
```

Comment:
Yeah this is duplicating logic is all, should be fine otherwise.
Just change the `if(M_HARDCORE in mutations)` to `if(M_HARDCORE in mutations && !miming)`